plugin/src/helpers/config.ts

```diff
@@ -115,6 +116,27 @@ export async function checkConfig({ utils, netlifyConfig }): Promise<void> {
   }
 }
 
+/**
+ * Copies the contents of the Gatsby datastore file to the public directory in order
+ * to be uploaded to the CDN.
+ *
+ * @param publishDir
+ */
+export async function createDatastoreMetadataFile(
+  publishDir: string,
+): Promise<void> {
+  const data = join(getGatsbyRoot(publishDir), '.cache/data/datastore/data.mdb')
+  const uniqueDataFileName = `data-${uuidv4()}.mdb`
+
+  ensureFileSync(`${publishDir}/${uniqueDataFileName}`)
+  copySync(data, `${publishDir}/${uniqueDataFileName}`)
+
+  const payload = { fileName: uniqueDataFileName, url: process.env.URL }
```

Comment:
As an alternative to using `process.env.URL` here, you could save just the path, and then in the lambda resolve it relative to `event.rawUrl`